### PR TITLE
[Form] Static call TimezoneType::getTimezones

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 3.2
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no 
| Tests pass?   | yes
| Fixed tickets | #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Spotted in #23648

### CODE REVIEW

**Review by ogizanagi — APPROVED** [review]

**Comment by ogizanagi** [comment]
Seems like you've made fabbot unhappy though. It has feelings you know.
(You should really use the full PR template, even for minor patches.)

File: src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ public function loadChoiceList($value = null)
             return $this->choiceList;
         }
 
-        return $this->choiceList = new ArrayChoiceList($this->getTimezones(), $value);
+        return $this->choiceList = new ArrayChoiceList(self::getTimezones(), $value);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: Technically, the previous code allowed extending the class to provide alternative implem via inheritance. Was it on purpose? Can't say right now, but that should be considered before approval.

Comment (ro0NL) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: As of 3.0 getTimezones is a private static.. so not open for inheritance :)

Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: ok, strange, but verified: https://3v4l.org/n61o4

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php (hunk 1 above)
Comment (ro0NL) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: Ah like so.. you're right i didnt checked it.. i assumed it :)

Comment (nicolas-grekas) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: OH, but you can turn it protected, so still open for extension!
https://3v4l.org/f2STp

Comment (ro0NL) [inline src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php]: Right.I cant imagine this is done on purpose.. at least core doesnt seem to rely on it elsewhere.

Nice catch btw :)

**Comment by ogizanagi** [comment]
Thanks @ro0NL.

**Pull request merged**